tutorials/test_status/regex_on_invailds.yml

```diff
@@ -0,0 +1,11 @@
+buildspecs:
+  regex_on_invalid_fname:
+    type: script
+    executor: generic.local.bash
+    description: Test regex on an invalid file name
+    run: |
+      echo "Hello" > hello.txt
+    status:
+      file_regex:
+      - file: /fox.txt
+      - exp: '^(Hello)$'
```

Comment:
@braeeast2001 good job this looks good now can you add one more test in `regex_on_invalids.yml` that will perform regular expression on valid file but regex is invalid.

Also there is a spelling error in name of file `regex_on_invailds.yml` -> `regex_on_invalids.yml`